### DISCUSSION TITLE
How to take a still photo at full resolution

### DISCUSSION THREAD

**Discussion by ivanmicetic** [geerlingguy/pi-webcam#35] [Q&A]
Is it possible to take a full resolution still picture when in USB webcam mode? Or do we have to disable some service before taking one?

**Comment by techydude** [comment]
have you tried the built-in raspistill util?

**Reply by ivanmicetic** [reply]
Yes, raspistill works but I need to interrupt the live view from the webcam. I was wandering if there was a way to interrupt (or pause) the video stream, save a picture in full resolution and then resume with the stream.
